COMPATIBILITY.md

```diff
@@ -87,7 +91,7 @@ Entries list the class name serving as the entry point to the relevant functiona
 - [X] `ec2`: known to work as is
```

Comment:
What if we used: `1.7+ of Workflow Plugin` instead. This implies support for this plugin was added in 1.7 and beyond of the workflow plugin.